drivers/ethernet/eth_mcux.c

```diff
@@ -932,6 +949,38 @@ static void eth_mcux_ptp_isr(void *p)
 }
 #endif
 
+#ifdef CONFIG_SOC_SERIES_IMX_RT
+static void eth_mcux_dispacher_isr(void *p)
+{
+	struct device *dev = p;
+	struct eth_context *context = dev->driver_data;
+	uint32_t EIR = ENET_GetInterruptStatus(ENET);
+	int irq_lock_key = irq_lock();
+
+
+	if (EIR & (kENET_RxBufferInterrupt | kENET_RxFrameInterrupt))
```

Comment:
This does not follow Zephyr coding style. Please place { in the same line as the if() statement like this
```
if (xxx) {

} else if () {

} else {

}
```